extensions/gc/service.js

```diff
@@ -39,22 +55,22 @@ function initAndStart() {
             return;
         }
         logger.info('management init done');
-        healthServer.onReadyCheck(log => {
-            if (garbageCollector.isReady()) {
-                return true;
-            }
-            log.error('GarbageCollector is not ready!');
-            return false;
-        });
-        logger.info('Starting HealthProbe server');
-        healthServer.start();
-        garbageCollector.start(err => {
+        startProbeServer(config.healthcheckServer, (err, probeServer) => {
```

Comment:
Not sure how much more work it is but using a new config object to match the others might make sense here

Reply:
new config object?

Reply:
yea like config.extensions.gc.probeServer to match the replication ones